### ISSUE TITLE
Upgrade to all f-strings

### ISSUE THREAD

**Issue by fubuloubu** [ethereum/vyper#1567]
### What's your issue about?
The code uses a mix of `%s`, `.format()`, and `f""`, which can be confusing which style we prefer.

### How can it be fixed?
Change it all to one style, preferably f-strings.

Also, document this in the style guide

**Comment by wschwab** [comment]
I'd like to take this. 2 questions:
1) When would you like to have this done by?
2) Where is the style guide?

**Comment by fubuloubu** [comment]
So, you can format a string literal as bytes via `.encode("utf-8")`:
```python
>>> "m12mm345mm".encode("utf-8")
b'm12mm345mm'
>>> "m12mm345mm{}".format("afgs").encode("utf-8")
b'm12mm345mmafgs'
>>> stuff = "afgafdg"
>>> f"m12mm345mm{stuff}".encode("utf-8")
b'm12mm345mmafgafdg'
```

Thank you for taking this on!

**Comment by fubuloubu** [comment]
As far as your questions, 1) whenever you can get it done by is fine, just set up a PR with "fixes #1567" in it (so it links back to this one) so we know what the progress is, and 2) not sure we have a style guide for how to write f-strings, if you have questions just ping us in the gitter. Thank you again!
